InputGroup/src/style.less

```diff
@@ -1,8 +1,9 @@
 .dnn-ui-common-input-group {
         position: relative;
-        display: table;
+        display: block;
         border-collapse: separate;
         width: 100%;
+        float: left;
```

Comment:
Are you sure `float` isn't going to cause trouble?

Reply:
I don't know about the code of each components. It's part of our next issues when we will decide how to manage common components

Reply:
I mean, this looks like a new styl prop here. I worry it could break some UI